UnityProject/Assets/Scripts/Managers/SoundManager.cs

```diff
@@ -675,12 +677,22 @@ private void PlaySource(SoundSpawn source, bool polyphonic = false, bool Global
 		if (!forceMixer)
 		{
 			if (!Global
-			    && PlayerManager.LocalPlayer != null
-			    && (MatrixManager.Linecast(PlayerManager.LocalPlayer.TileWorldPosition().To3Int(),
-					    LayerTypeSelection.Walls, layerMask, source.RegisterTile.WorldPositionClient.To2Int().To3Int())
-				    .ItHit))
+			    && PlayerManager.LocalPlayer != null)
 			{
-				source.AudioSource.outputAudioMixerGroup = soundManager.MuffledMixer;
+				if ((PlayerManager.LocalPlayer.TileWorldPosition().To3Int() -  source.RegisterTile.WorldPositionClient.To2Int().To3Int()).magnitude < 50)
```

Comment:
Vector2.Distance() little bit cleaner and removes all those To3Int and To2Int

Reply:
ok